cms/static/js/programs/routers/core_app_program_router.js

```diff
@@ -0,0 +1,67 @@
+define([
```

Comment:
@rlucioni I agree that we should rename it and that it doesn't really make sense to have a router folder when there is only 1 router.